subprojects/gradle/build-checks/src/main/kotlin/com/avito/android/build_checks/internal/incremental_kapt/RoomIncrementalKaptChecker.kt

```diff
@@ -1,80 +0,0 @@
-package com.avito.android.build_checks.internal.incremental_kapt
-
-import com.avito.kotlin.dsl.withType
-import com.avito.logger.LoggerFactory
-import com.avito.logger.create
-import org.gradle.api.JavaVersion
-import org.gradle.api.Project
-import org.jetbrains.kotlin.gradle.internal.KaptGenerateStubsTask
-import java.net.URLClassLoader
-import javax.annotation.processing.AbstractProcessor
-
-internal class RoomIncrementalKaptChecker(
```

Comment:
> IncrementalKaptChecks removed
> heavy subprojects call, implemented as part of room convention

I didn't find equivalent check logic in room convention plugin.

Reply:
I see, it's in another PR. 

I propose to have it in this plugin because of:
- Integration tests
- OS useful plugin

Reply:
it should be separate plugin, because it is intended to be applied to modules, not a root project

Reply:
Discussed that we can remove this check due to Java 11 wider adoption.